src/main/java/com/singh/astha/notificationservice/config/JwtAuthenticationHandler.java

```diff
@@ -27,30 +28,43 @@ public JwtAuthenticationHandler(JwtService jwtService) {
     }
 
     @Override
-    public void commence(HttpServletRequest request, HttpServletResponse response,
-                         AuthenticationException authException) throws IOException {
+    public void commence(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            AuthenticationException authException
+    ) throws IOException {
         response.setHeader(Constants.CONTENT_TYPE, Constants.APPLICATION_JSON);
         try {
             String authorizationHeader = request.getHeader(Constants.AUTHORIZATION);
             if (authorizationHeader == null) {
-                writeResponse(response, HttpStatus.BAD_REQUEST.value(),
+                writeResponse(
+                        response,
+                        HttpStatus.BAD_REQUEST.value(),
                         ResponseWrapper.failure(null, ErrorMessages.AUTHORIZATION_HEADER_MUST_BE_PRESENT)
                 );
                 return;
             }
             jwtService.verifyAndDecodeToken(authorizationHeader);
         } catch (ResponseException responseException) {
-            writeResponse(response, responseException.getStatus().value(),
+            writeResponse(
+                    response,
+                    responseException.getStatus().value(),
                     ResponseWrapper.failure(responseException.getPayload(), responseException.getMessage())
             );
             return;
         }
-        writeResponse(response, HttpStatus.FORBIDDEN.value(),
-                ResponseWrapper.failure(null, ErrorMessages.ACCESS_DENIED));
+        writeResponse(
+                response,
+                HttpStatus.FORBIDDEN.value(),
+                ResponseWrapper.failure(null, ErrorMessages.ACCESS_DENIED)
+        );
```

Comment:
The assumption of 403 here is not correct. But as this is a Spotless PR, create a new issue to fix this in the future.

Reply:
Created a separate Issue for this --> https://github.com/astha-singh-2445/Notification_Service_Spring_App/issues/19

Reply:
Created Issue-> https://github.com/astha-singh-2445/Notification_Service_Spring_App/issues/20